src/open_inwoner/templates/pages/plans/list.html

```diff
@@ -1,11 +1,48 @@
 {% extends 'master.html' %}
-{% load i18n button_tags card_tags map_tags utils string_tags %}
+{% load i18n button_tags string_tags pagination_tags helpers %}
 
 {% block content %}
     <h1 class="h1">
         {% trans "Samenwerken" %}
         {% button href="plans:plan_create" text=_("Start nieuwe samenwerking") primary=True icon="group" icon_outlined=True %}
     </h1>
     {% optional_paragraph configurable_text.plans_page.plans_intro %}
-    {% card_container plans=object_list columns=2 %}
+    
```

Comment:
Note that the userstory is about having listview as an expanded table when the User.contact_type is Begeleider. For regular users the card-listview is fine.

Reply:
Yes indeed, but in my notes from the weekly I had that we would go for one list view. I will change this according to the user.

Reply:
@alextreme discussed this with the PO and we will go for two options, one for the regular users (old list view) and one for the 'begeleider' (the updated one).